extensions/hexpm/src/search-packages.tsx

```diff
@@ -0,0 +1,205 @@
+import {
+  ActionPanel,
+  Color,
+  Icon,
+  List,
+  showToast,
+  useNavigation,
+  Action,
+  Toast,
+  getPreferenceValues,
+} from "@raycast/api";
+import { useState } from "react";
+import { Package } from "./types";
+import { useDebounce } from "use-debounce";
+import { usePackagesQuery } from "./hooks/use-packages-query";
+import { useVisitedPackages } from "./hooks/use-visited-packages";
+import { useFrecencySorting } from "@raycast/utils";
+import { packageId } from "./helpers";
+
+export default function SearchPackagesCommand() {
+  const preferenceValues = getPreferenceValues();
+  const [searchText, setSearchText] = useState<string>("");
+  const [debouncedSearchText] = useDebounce(searchText, 500);
```

Comment:
```suggestion
  const [debouncedSearchText] = useDebounce(searchText, 500);
```

Should I remove this `useDebounce` hook?